tests/charts/log_groomer.py

```diff
@@ -90,6 +90,28 @@ def test_log_groomer_collector_default_retention_days(self):
         )
         assert jmespath.search("spec.template.spec.containers[1].env[0].value", docs[0]) == "15"
 
+    def test_log_groomer_collector_custom_env(self):
+        env = [
+            {"name": "WORKER_RELEASE_NAME", "value": "{{ .Release.Name }}-workers"},
+            {"name": "AIRFLOW__LOG_RETENTION_DAYS", "value": "5"},
```

Comment:
Let's use a different env var, or drop this one altogether (that'd be my vote) - it's exposed by the values file already and tested in `test_log_groomer_retention_days_overrides`.

Reply:
@jed can you check now i have added all the necessary changes now